src/readme.js

```diff
@@ -0,0 +1,199 @@
+const buildTitleAndBadge = ({ title, license }) => `# ${title}
+![${license}](https://img.shields.io/static/v1?label=${license}&message=License&color=green)`;
+
+const buildTableOfContents = ({ installationIncluded, testIncluded }) => {
```

Comment:
This function needs to be refactored majorly. What if someone doesn't want to include a user story? What if we don't include questions? etc.  Currently this function is very specific and would not be reusable for other scenarios.

How can we pass it the whole object of answers, go through the object to check each key to see if the value is truthy and then only add the section if it exists? What kind of method would we need here?

You want to avoid being as specific as you have here, and avoid repetitive code. Although both methods would produce the same result, if you can do it in 10 lines, rather than the 55 lines (!) you have used to achieve the same thing, I'm sure you can tell which would be nicer 😃 

Have a think about it and drop me a message if you want any help/tips.